src/GLHelpers.cpp

```diff
@@ -87,9 +84,22 @@ void ErrorCallback(int error, const char* description)
 	std::cerr << description << std::endl;
 }
 
-// Handles keyboard input events
-void KeyCallback(GLFWwindow* window, int key, int scancode, int action, int mods)
+
+
+// GLM DEBUG
+void PrintMat4(glm::mat4 const &m)
+{
+	for(int i = 0; i < 4; i++)
+	{
+		for(int j = 0; j< 4; j++)
```

Comment:
[Not important] minor inconsistent spacing between j and <

Reply:
d